src/components/Wiki/WikiCreate.tsx

```diff
@@ -0,0 +1,114 @@
+import MDEditor from '@uiw/react-md-editor';
+import { create, update } from 'apis/Wiki';
+import { DocumentData } from 'firebase/firestore';
+import { useState } from 'react';
+import styled from 'styled-components';
+import { media } from 'styles/media';
+
+interface createProps {
+  setIsEdit: React.Dispatch<React.SetStateAction<boolean>>;
+  selectedCategory: string;
+  data: DocumentData | undefined;
+}
+
+function WikiCreate({ setIsEdit, selectedCategory, data }: createProps) {
+  const isCreate = data === undefined;
```

Comment:
내려온 data 가 없을 때 "생성" 이다, 라는 의미인거죠?
이런 플래그 값을 만들때는 memoization을 하면 좋습니다.(성능면)
```suggestion
import isNil from 'lodash/isNil'

  const isCreateMode = useMemo(() => isNil(data), [data]);
```